### ISSUE TITLE
Realurl dependency

### ISSUE THREAD

**Issue by hasenbalg** [cundd/rest#45]
Hi, the extension was not working for me (Typo3 8.7) until I installed realurl. It is neither mentioned in the dependencies nor in the documentation.

**Comment by cundd** [comment]
Did you also configure the rewriting in .htaccess or you nginx configuration?

**Comment by cundd** [comment]
Actually the extension does not depend on RealURL, but the URL rewriting has to be configured properly - this may be missing in the docs

**Comment by hasenbalg** [comment]
Thank you! What does 'configured properly' mean? I am not very familiar with .htaccess or nginx URL rewriting. I'm using the default .htaccess provided by TYPO3 8.7 and I located the block concerning URL rewriting. Must the new rule look something like `RewriteRule    ^rest/?$    XXX.php`?

**Comment by cundd** [comment]
I think the core part to configure the rewriting in `.htaccess` are the following lines:

```
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FILENAME} !-d
RewriteCond %{REQUEST_FILENAME} !-l
RewriteRule ^.*$ %{ENV:CWD}index.php [QSA,L]
```

They are already part to the default TYPO3 `_.htaccess` file.
Nothing else should be required for the REST extension to work.

Could it be that you renamed/copied `_.htaccess` to `.htaccess` when you installed RealURL?

**Comment by hasenbalg** [comment]
Thank very much for your help! I reinstalled my TYPO3 instance  and the REST extension to isolate the problem. The extension did not work, like before. Now I installed RealURL (I forgot to configure) and it worked. After uninstalling RealURL it keept working. My college found the line `'clearcacheonload' => true,` in `ext_emconf.php` of RealURL which must have **cleared all caches**. So documentation is OK and the dependencies too.
